specs/pvs/corla.pvs

```diff
@@ -922,8 +995,10 @@ END cast_vote_records
 %%% the county-level, and one is for audit board members.
```

Comment:
I've refined the datatype to include the public interface.  The interface through which those involved in the audit input ballot interpretations is, conceptually, distinct from the top-level county interface, which is why they are modeled as two different UIs.